kyuubi-common/src/main/scala/org/apache/kyuubi/session/SessionManager.scala

```diff
@@ -198,6 +198,8 @@ abstract class SessionManager(name: String) extends CompositeService(name) {
     }
   }
 
+  def isShutdown: Boolean = shutdown
```

Comment:
IMO, here is unnecessary. You could directly use the `shutdown` to check.

Reply:
`shutdown` is a private mutable field, it should be good to add a method here